css/contact.css

```diff
@@ -35,16 +36,16 @@ textarea {
 	background-color: var(--color-white);
 	border: 1px solid #6c6c6c;
 	width: 100%;
-	border-radius: 5px;
+	border-radius: var(--spacing10);
 	padding: var(--spacing10);
-	margin: 5px 0;
+	/* margin: 5px 0; */
```

Comment:
Some commented out code! please remove it when you get a chance to keep code clean

Reply:
comment removed

Reply:
Astounding